test/sass/scss/scss_test.rb

```diff
@@ -4221,4 +4221,70 @@ def test_crlf
 SCSS
   end
 
+  def test_dynamic_include
```

Comment:
Sure. I'll remove these tests once the sass-spec bits land, but would like to keep them here until after the merge.

I'm not loving this workflow and I need to think of a way for sass-spec tests to be more seamlessly integrated into the new feature work we do. One idea is that we can have a directory where we add new sass-spec tests and those test can be ran by the sass-spec runner even though they are not in the sass-spec repo. Then after a feature lands in ruby-sass we can migrate the tests to sass-spec.